src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolFinalStep.tsx

```diff
@@ -55,32 +47,25 @@ const TableToolFinalStep = ({
   );
   const publication = fullPublication?.publication;
 
-  const handlePermalinkClick = async () => {
-    if (!currentTableHeaders) {
-      return;
+  const getMethodologyLink = () => {
+    if (publication?.methodology) {
+      return (
+        <Link
+          to="/methodology/[methodology]"
+          as={`/methodology/${publication.methodology.slug}`}
+        >
+          {publication.methodology.title}
+        </Link>
+      );
     }
-    setPermalinkLoading(true);
-
-    const { id } = await permalinkService.createPermalink(
-      {
-        query,
-        configuration: {
-          tableHeaders: mapUnmappedTableHeaders(currentTableHeaders),
-        },
-      },
-      selectedPublication.selectedRelease.id,
-    );
-
-    setPermalinkId(id);
-    setPermalinkLoading(false);
-  };
-
-  const handleCopyClick = () => {
-    const el = document.querySelector(
-      "[data-testid='permalink-generated-url']",
-    ) as HTMLInputElement;
-    el?.select();
-    document.execCommand('copy');
+    if (publication?.externalMethodology) {
+      return (
+        <a href={publication.externalMethodology.url}>
```

Comment:
Probably a wider discussion, but the govuk advice is to avoid opening links in new tabs https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab

Reply:
Ah good point, haven't seen this advice before :smile:

Would still suggest using `Link` to be consistent as it still handles absolute links as well.